app/build.gradle

```diff
@@ -36,4 +36,5 @@ dependencies {
     testImplementation 'junit:junit:4.+'
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    implementation 'com.google.android.gms:play-services-location:18.0.0'
```

Comment:
move this line to upper implemetation items. Remove newline at the start of the block